### PR TITLE
Update CT Mapper's documentation

### PR DESCRIPTION
**Pull request by codingllama**
* Added tree type and hash strategy to createtree
* Use correct ports on commands (RPC, not HTTP)
* Use ${tree_id} consistently, print the ID in the console

### CODE REVIEW

**Comment by codingllama** [comment]
Issue #901

**Comment by codingllama** [comment]
@gdbelvin @phad: friendly ping.

File: examples/ct/ctmapper/README.md [hunk 1]
```diff
@@ -25,20 +25,24 @@ go build ./examples/ct/ctmapper/lookup
 go build ./cmd/createtree/
 tree_id=$(./createtree \
```
Comment (gdbelvin) [inline examples/ct/ctmapper/README.md]: IIRC `./createtree` returns a whole json structure.  In KT, we used [jq](https://stedolan.github.io/jq/) to parse the output and obtain the `tree_id`.  Is something similar needed here?

Comment (phad) [reply examples/ct/ctmapper/README.md]: No, it doesn't - I just tried it and the output was simply the tree ID:

$ ./createtree  --admin_server=localhost:8090 --tree_type=MAP --hash_strategy=TEST_MAP_HASHER --signature_algorithm=ECDSA
9216245476404281793

Comment (phad) [reply examples/ct/ctmapper/README.md]: Maybe it used to output JSON?

Comment (codingllama) [reply examples/ct/ctmapper/README.md]: It prints only the tree ID. It's explicitly meant to be used this way.

https://github.com/google/trillian/blob/master/cmd/createtree/main.go#L21
https://github.com/google/trillian/blob/master/cmd/createtree/main.go#L179.

File: examples/ct/ctmapper/README.md [hunk 2]
```diff
@@ -25,20 +25,24 @@ go build ./examples/ct/ctmapper/lookup
...
     --admin_server=localhost:8090 \
-    --signature_algorithm=ECDSA)
+    --hash_strategy TEST_MAP_HASHER \
```
Comment (phad) [inline examples/ct/ctmapper/README.md]: nit: be consistent with the other flags and use '='

Comment (codingllama) [reply examples/ct/ctmapper/README.md]: Done, well spotted.

**Pull request merged**